inst/tests/other.Rraw

```diff
@@ -781,3 +781,297 @@ if (loaded[["vctrs"]]) {
   DT = data.table(a = 1, b = list_of(mtcars))
   test(33, DT, output="<vctrs_list_of>.*<data\\.frame\\[32x11\\]>")
 }
+
+# NB: currently, RSQLite requires DBI, so partially redundant, but future-proof.
+if (loaded[["DBI"]] && loaded[["RSQLite"]]) {
+  # mergelist join tester vs SQLite, based on v1.9.8 non-equi join tester
```

Comment:
Finally I can drop my 9 years old join-tester-dev branch in my repo that I was keeping just no to have this piece of work gone :)